grafana/provisioning/dashboards/tvl_vaults.json

```diff
@@ -1706,12 +1733,12 @@
     "list": []
   },
   "time": {
-    "from": "now-24h",
+    "from": "now-5m",
```

Comment:
@DarkGhost7 is that intended to change default time range to the last 5 minutes instead of the last 24 hours like before ?

Reply:
fixed